src/h3/geo.pyx

```diff
@@ -112,6 +177,56 @@ def polyfill(geos, int res):
     return mv
 
 
+def polyfill_geojson(geojson, int res=9):
+    """ Set of hexagons whose center is contained in a GeoJson Polygon object.
+
+    The polygon is defined exactly as in the GeoJson standard, so
+    `geojson` should be a dictionary like:
+    {
+        'type': 'Polygon',
+        'coordinates': [...]
+    }
+
+    'coordinates' should be a list of LinearRings, where the first ring describes
+    the exterior boundary of the Polygon, and any subsequent LinearRings
+    describe holes in the polygon.
+
+    Note that we don't provide an option for the order of the coordinates,
+    as the GeoJson standard requires them to be in lng/lat order.
+
+    Parameters
+    ----------
+    geojson : dict
+    res : int
+        The resolution of the output hexagons
+    """
+
+    # todo: this one could handle multipolygons...
+
+    if geojson['type'] != 'Polygon':
+        raise ValueError('Only Polygon GeoJSON supported')
+
+    coords = geojson['coordinates']
+
+    out = polyfill_polygon(coords[0], holes=coords[1:], res=res, order='lnglat')
+
+    return out
+
+
+
+
+
+def polyfill(geojson, int res, geo_json_conformant=False):
+    """ Light wrapper around `polyfill_geojson` to provide backward compatibility.
+    """
+
+    gj = geojson.copy()
```

Comment:
This could be expensive for large input polygons, right? Two thoughts:
- Minimally, move the `copy` operation inside `if not geo_json_conformant`, probably with two separate `return` statements
- Ideally, pass the `geojson_conformant` param down to `make_geofence` and swap the input args to `geo2coord` appropriately. This is what we do in JS - you just change the way you pass the memory to C, you never have to duplicate the input.

Reply:
Good point!

Resolved this so that it no longer does the copy. While I was at it, I decided that having the `order='latlng'` keyword param was kind of awkward, and it should really be a boolean param. I added `lnglat_order=False` instead. Let me know what you all think.